projects/element-ng/chat-messages/si-markdown-content.component.ts

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { CommonModule } from '@angular/common';
+import { Component, computed, inject, input, SecurityContext } from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+/** @experimental */
+@Component({
+  selector: 'si-markdown-content',
+  imports: [CommonModule],
+  templateUrl: './si-markdown-content.component.html',
+  styleUrl: './si-markdown-content.component.scss'
+})
+export class SiMarkdownContentComponent {
```

Comment:
Why do we use our own md renderer?

I think it would be nicer to have an API for this and then we can just ship one implementation based on a OSS project.